src/karts/skidding.cpp

```diff
@@ -47,8 +47,7 @@ Skidding::Skidding(Kart *kart, const SkiddingProperties *sp)
     m_actual_curve->setVisible(false);
 #endif
     m_kart = kart;
-    copyFrom(sp);
-    m_skid_reduce_turn_delta = m_skid_reduce_turn_max - m_skid_reduce_turn_min;
+    m_skid_reduce_turn_delta = m_kart->getKartProperties()->getSkidReduceTurnMax() - m_kart->getKartProperties()->getSkidReduceTurnMin();
```

Comment:
General remark, that's not really bad but a stylistic remark. Many of these lines are quite long and repeat "m_kart->getKartProperties()" over and over. It may be better to create a "kart_properties" variable at the start, calling "kart_properties = m_kart->getKartProperties()" only once and then avoid the repeated calls